static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -66,14 +66,22 @@ require.def(
                 SUBTITLE : 19,
                 CUSTOM : 20
             },
+            PlayerEmps: {
```

Comment:
Actually this name comes directly from platform API, where SEF on "Open" method initializes some EMP or module if your prefer.